DB/src/main/java/io/deephaven/db/v2/locations/local/ReadOnlyParquetTableLocation.java

```diff
@@ -11,6 +11,7 @@
 import io.deephaven.db.v2.locations.parquet.topage.*;
 import io.deephaven.db.v2.parquet.ParquetTableWriter;
 import io.deephaven.db.v2.sources.chunk.Attributes;
+import io.deephaven.util.codec.ByteArrayCodec;
```

Comment:
This is unused, but I think it should be used; you're using the fully-qualified name on L212.

Reply:
He's dead, Jim.